### PR TITLE
Add a new target: mbed 6LoWPAN Border Router HAT

### PR DESCRIPTION
**Pull request by devran01**
Signed-off-by: Devaraj Ranganna <devaraj.ranganna@arm.com>

### CODE REVIEW

**Comment by devran01** [comment]
Should fix the issue https://github.com/mbedmicro/pyOCD/issues/266

File: pyOCD/board/mbed_board.py [hunk 1]
```diff
@@ -87,6 +87,7 @@ def __init__(self, name, target, binary):
                 "9012": BoardInfo(  "Seeed Tiny BLE",       "nrf51",            "l1_nrf51.bin",         ),
                 "9900": BoardInfo(  "Microbit",             "nrf51",            "l1_microbit.bin",      ),
                 "C004": BoardInfo(  "tinyK20",              "k20d50m",          "l1_k20d50m.bin",       ),
+                "7402": BoardInfo(  "FRDM-K64F",            "k64f",             "l1_k64f.bin",          ),
```
Comment (c1728p9) [inline pyOCD/board/mbed_board.py]: Please order this by board ID. Also, you'll want to change the board name to reflect that it is a "mbed 6LoWPAN Border Router HAT".

**Comment by devran01** [comment]
@c1728p9 Thanks for the tips. Updated as suggested.

File: pyOCD/board/mbed_board.py [hunk 2]
```diff
@@ -82,6 +82,7 @@ def __init__(self, name, target, binary):
                 "1600": BoardInfo(  "Bambino 210",          "lpc4330",          "l1_lpc4330.bin",       ),
                 "1605": BoardInfo(  "Bambino 210E",         "lpc4330",          "l1_lpc4330.bin",       ),
                 "2201": BoardInfo(  "WIZwik_W7500",         "w7500",            "l1_w7500mbed.bin",     ),
+                "7402": BoardInfo(  "mbed 6LoWPAN Border Router HAT",            "k64f",             "l1_k64f.bin",          ),
```
Comment (c1728p9) [inline pyOCD/board/mbed_board.py]: Nitpick - can the spacing be adjusted so the ending bracked closes where the rest do?

**Comment by devran01** [comment]
@c1728p9 Fixed.

**Comment by studavekar** [comment]
Can one of the admins verify this patch?

**Pull request merged**